indexing-service/src/test/java/org/apache/druid/indexing/overlord/TaskLockboxTest.java

```diff
@@ -1258,6 +1259,39 @@ public void testGetLockedIntervalsForRevokedLocks() throws Exception
     );
   }
 
+  @Test
+  public void testFailedToReacquireTaskLock() throws Exception
+  {
+    final Task badTask0 = NoopTask.withGroupId("BadTask");
+    final Task badTask1 = NoopTask.withGroupId("BadTask");
+    final Task goodTask0 = NoopTask.withGroupId("GoodTask");
+    taskStorage.insert(badTask0, TaskStatus.running(badTask0.getId()));
+    taskStorage.insert(badTask1, TaskStatus.running(badTask1.getId()));
+    taskStorage.insert(goodTask0, TaskStatus.running(goodTask0.getId()));
+
+    TaskLockbox testLockbox = new NullLockPosseTaskLockbox(taskStorage, metadataStorageCoordinator);
+    testLockbox.add(badTask0);
+    testLockbox.add(badTask1);
+    testLockbox.add(goodTask0);
+
+    testLockbox.tryLock(badTask0, new TimeChunkLockRequest(TaskLockType.EXCLUSIVE,
```

Comment:
Nit: Style: Put each arg on a separate line (when necessary) for consistency with the rest of Druid code.

Reply:
Done